src/app/src/actions/logDownload.js

```diff
@@ -0,0 +1,32 @@
+import { createAction } from 'redux-act';
+
+import apiRequest from '../util/apiRequest';
+
+import { makeLogDownloadUrl, logErrorAndDispatchFailure } from '../util/util';
+
+export const startLogDownload =
+    createAction('START_LOG_DOWNLOAD');
+export const failLogDownload =
+    createAction('FAIL_LOG_DOWNLOAD');
+export const completeLogDownload =
+    createAction('COMPLETE_LOG_DOWNLOAD');
+
+export function logDownload(rows, callback) {
+    return (dispatch) => {
```

Comment:
Since the facilities data lives in the app's Redux store state, conceivably we could change this function to actually perform the download directly rather than having to pass `rows` and `callback` in as arguments. If we handled it this way, we could get the facilities data here via `getState` and then directly import the `downloadFacilitiesCSV` utility function to execute it in the promise chain.

Reply:
We discussed this in person and it does make sense given the future refactoring of the search result handling to have this action fetch the results from the state rather than having the caller pass them in.

Reply:
Implemented in fixup c53103b